Formula/s/stellar-cli.rb

```diff
@@ -0,0 +1,32 @@
+class StellarCli < Formula
+  desc "Stellar command-line tool for interacting with the Stellar network"
+  homepage "https://developers.stellar.org"
+  url "https://github.com/stellar/stellar-cli.git",
+      tag:      "v21.3.0",
+      revision: "11746937e813346272faf68e70cbdbc1b0df780f"
+  license "Apache-2.0"
+  head "https://github.com/stellar/stellar-cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    chdir "cmd/stellar-cli" do
+      system "cargo", "install", "--bin=stellar", "--features=opt", *std_cargo_args
+    end
+  end
+
+  test do
+    assert_match "stellar 21.3.0", shell_output("#{bin}/stellar version")
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/stellar version")
```
Let's avoid having to change this each time the version is updated, and prevent having it break if the format of `stellar version` changes.